src/WebJobs.Script/Description/FSharp/FSharpCompiler.cs

```diff
@@ -115,11 +117,11 @@ public ICompilation GetFunctionCompilation(FunctionMetadata functionMetadata)
                 }
 
                 // This output DLL isn't actually written by FSharp.Compiler.Service when CompileToDynamicAssembly is called
-                otherFlags.Add("--out:" + Path.ChangeExtension(Path.GetTempFileName(), "exe"));
+                otherFlags.Add("--out:" + Path.ChangeExtension(Path.GetTempFileName(), "dll"));
```

Comment:
Changed from `exe` to `dll` hoping FS0988 would go away, but it didn't. Kept it as `dll` as it seems appropriate.